### PR TITLE
bugfix/meeting-and-person-voting-tables

### LINKED ISSUE #155: Voting tables are not rendering all the data
### Describe the Bug

Voting tables are not rendering all the data. 

In the person's page, `Meeting` have no data displayed and `Council Decision` should be these [constants](https://github.com/CouncilDataProject/cdp-backend/blob/main/cdp_backend/database/constants.py#L10) -- Passed or Failed
image

Similarly, in the `Votes` tab of the the event page, the `Council Decision` column should have value Passed or Failed
image
(side note: These legislation descriptions can be too long for a cell, we should use top 5 keywords or topics to describe the legislation instead)

### PR DESCRIPTION
**Pull request by BrianL3**
### Link to Relevant Issue

This pull request resolves #155 

### Description of Changes

Fixes outstanding issues with Voting Tables: the right hand side column was not rendering.

### CODE REVIEW

File: src/assets/strings/en.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ const en = {
   transcript: "Transcript",
   agenda: "Agenda",
   session: "Session",
+  passed: "Passed",
```
Comment (tohuynh) [inline src/assets/strings/en.ts:55]: Add "Failed"

File: src/assets/strings/es.ts [hunk 2]
```diff
@@ -52,6 +52,7 @@ const es = {
   transcript: "Transcripción",
   agenda: "Agenda",
   session: "Sesión",
+  passed: "Passed",
```
Comment (tohuynh) [inline src/assets/strings/es.ts:55]: Add "Failed"

Add "Passed" and "Failed" for German too

Add `passed: string` and `failed: string` to src/assets/LocalisedString.tsx

File: src/components/Shared/DecisionResult.tsx [hunk 3]
```diff
@@ -35,6 +38,9 @@ const DecisionResult: FC<DecisionResultProps> = ({ result }: DecisionResultProps
     case MATTER_STATUS_DECISION.IN_PROGRESS:
       statusIcon = <InProgressIcon />;
       break;
+    case EVENT_MINUTES_ITEM_DECISION.PASSED:
+      statusIcon = <AdoptedIcon />;
+      break;
```
Comment (tohuynh) [inline src/components/Shared/DecisionResult.tsx:43]: This switch control could be shorten using fall through:
```
switch (result) {
    case EVENT_MINUTES_ITEM_DECISION.PASSED:
    case MATTER_STATUS_DECISION.ADOPTED:
    case VOTE_DECISION.APPROVE:
      statusIcon = <AdoptedIcon />;
      break;
    case EVENT_MINUTES_ITEM_DECISION.FAILED:
    case MATTER_STATUS_DECISION.REJECTED:  
    case VOTE_DECISION.REJECT:
      statusIcon = <RejectedIcon />;
      break;
    case VOTE_DECISION.ABSTAIN:
      statusIcon = <AbstainIcon />;
      break;
    case MATTER_STATUS_DECISION.IN_PROGRESS:
      statusIcon = <InProgressIcon />;
      break;
    default:
      break;
  }
```

File: src/components/Tables/VotingTable/VotingTable.tsx [hunk 4]
```diff
@@ -47,12 +47,13 @@ const VotingTable = ({ name, votesPage }: VotingTableProps) => {
     if (!vote.matter || !vote.decision || !vote.event) return renderEmpty(index);
     const legislationName = vote.matter.name;
     const voteDecision = vote.decision;
-    const councilDecision = vote.council_decision;
+    console.log(`What is vote: ${JSON.stringify(vote)}`);
```
Comment (tohuynh) [inline src/components/Tables/VotingTable/VotingTable.tsx:50]: Remove unnecessary console.log

File: src/constants/ProjectConstants.ts [hunk 5]
```diff
@@ -10,6 +10,12 @@ export enum MATTER_STATUS_DECISION {
   IN_PROGRESS = "In Progress",
 }
 
+export enum EVENT_MINUTES_ITEM_DECISION {
+  PASSED = "Passed",
+  REJECTED = "Rejected",
+  IN_PROGRESS = "In Progress",
```
Comment (tohuynh) [inline src/constants/ProjectConstants.ts:16]: This enum should have only "Passed" and "Failed"

**Review by tohuynh — CHANGES_REQUESTED** [review]
Looks good. Fixes the missing data in tables, except for the "Failed" case because there is no "failed" in `strings`.

Have some minor asks in the comments.

Could you also change all the `council_decision`s in MeetingVotesTable.stories.tsx to have "Passed" or "Failed"? And `councilDecision` in VotingTableRow.stories.tsx to have "Passed" or "Failed"?

**Comment by BrianL3** [comment]
> Could you also change all the council_decisions in MeetingVotesTable.stories.tsx to have "Passed" or "Failed"? And councilDecision in VotingTableRow.stories.tsx to have "Passed" or "Failed"?

I think we should re-write these components so they have "real" models that they rely on for their prop types. MeetingVote and IndividualMeetingVote are confusing to me.

**Pull request merged**